api/v1alpha1/componentv2_types.go

```diff
@@ -24,7 +24,7 @@ type ComponentV2Spec struct {
 	Type ComponentType `json:"type"`
 
 	// Build defines the build configuration for the component
-	Build BuildSpecInComponent `json:"build"`
+	Build BuildSpecInComponent `json:"build,omitempty"`
```

Comment:
Have you rebased your changes from https://github.com/openchoreo/openchoreo/pull/311?
Looks like there are two commits from that PR is duplicated